ui/lib/core/addon/components/info-table-row.js

```diff
@@ -63,4 +64,16 @@ export default Component.extend({
         return false;
     }
   }),
+
+  didInsertElement() {
+    this._super(...arguments);
+    const labelDiv = this.element.querySelector('div');
+    const labelText = this.element.querySelector('.is-label');
+    if (labelDiv && labelText) {
+      if (labelText.offsetWidth > labelDiv.offsetWidth) {
+        labelDiv.classList.add('label-overflow');
+        this.set('hasLabelOverflow', true);
+      }
```

Comment:
Nice work! `didInsertElement` is a good place for this. I'm wondering if it is worthwhile to add a test to check if the `label-overflow` class is applied to the element and that the tooltip appears for long labels?

Reply:
Thanks @zofskeez! I'd gone back and forth on a test..but I think that you're right to suggest it- this component shows up so many places it's probably worthwhile to include. I'll add!